tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -132,10 +129,10 @@ def GetUnvalidatedEntities(self) -> Dict[str, str]:
     }
 
   def GetExtraEntities(self) -> Dict[str, str]:
-    """Returns a mapping of entity_guid to entity_code
+    """Gets entities reported by in telemetry payload not in building config.
```

Comment:
typo: change to "reported in telemetry payload but not"

Reply:
Done.